src/core/getters/enum.ts

```diff
@@ -20,16 +20,7 @@ export const getEnumImplementation = (value: string, type: string) => {
     // nullable value shouldn't be in the enum implementation
     if (val === 'null') return acc;
     const isTypeNumber = type === 'number';
-    const isNumber = !Number.isNaN(Number(val.slice(1, -1)));
-    const key =
-      isNumber || isTypeNumber
-        ? toNumberKey(isTypeNumber ? val.toString() : val.slice(1, -1))
-        : sanitize(val, { underscore: '_', whitespace: '_', dash: '-' });
-
-    return (
-      acc +
-      `  ${keyword.isIdentifierNameES5(key) ? key : `'${key}'`}: ${val},\n`
-    );
+    return acc + ` ${isTypeNumber ? toNumberKey(val) : val}: ${val},\n`;
```

Comment:
you cannot simplify it like that. this implementation was there for some reason

Reply:
and it's also a breaking change

Reply:
In you case I think it's just that we need to tweak the sanitize to accept those characters

Reply:
something like this should do the job

<img width="1043" alt="Screen Shot 2022-07-19 at 12 51 39" src="https://user-images.githubusercontent.com/10516382/179733580-12606c33-37ec-4332-aa78-454c16f8cbec.png">